example/ck_tile/19_gemm_multi_d/utils.hpp

```diff
@@ -6,21 +6,13 @@
 struct MultiplyMultiply
 {
     template <typename E, typename C, typename D0, typename D1>
-    CK_TILE_DEVICE auto operator()(E& e, const C& c, const D0& d0, const D1& d1) const -> void
+    __host__ __device__ auto operator()(E& e, const C& c, const D0& d0, const D1& d1) const -> void
```

Comment:
You have to use CK_TILE_HOST_DEVICE since __host__ __device__ can cause redefinition in ck tile